### PR TITLE
Remove COS requirement while running e2e nvidia gpu tests.

### PR DESCRIPTION
**Pull request by rohitagarwal003**
```release-note
NONE
```

### CODE REVIEW

File: test/e2e/scheduling/nvidia-gpus.go [hunk 1]
```diff
@@ -69,15 +67,12 @@ func makeCudaAdditionDevicePluginTestPod() *v1.Pod {
 	return testPod
 }
 
-func isClusterRunningCOS(f *framework.Framework) bool {
+func logOSImages(f *framework.Framework) {
```
Comment (jiayingz) [inline test/e2e/scheduling/nvidia-gpus.go]: Do we also want to remove COS from testNvidiaGPUsOnCOS() function name and the test func description in line 177?

Comment (mindprince) [reply test/e2e/scheduling/nvidia-gpus.go]: Updated.

Comment (jiayingz) [reply test/e2e/scheduling/nvidia-gpus.go]: Maybe also remove "on Container Optimized OS only" on line 177?

Comment (mindprince) [reply test/e2e/scheduling/nvidia-gpus.go]: Done.

**Pull request merged**